packages/editor/src/components/post-actions/private-actions.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import { __ } from '@wordpress/i18n';
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import PostExcerpt from '../post-excerpt';
+import PluginPostExcerpt from '../post-excerpt/plugin';
+import { store as editorStore } from '../../store';
+
+export function useEditExcerptAction() {
```

Comment:
Finding the best path forward for consolidating the actions is also discussed [here](https://github.com/WordPress/gutenberg/pull/60754), but for now I added this private action and kind of 'separated' it from the other actions here..